moxin-frontend/src/shared/actions.rs

```diff
@@ -0,0 +1,9 @@
+use makepad_widgets::DefaultNone;
+use moxin_protocol::data::FileID;
+
+#[derive(Clone, DefaultNone, Debug)]
+pub enum DownloadedFileAction {
```

Comment:
since this action is now being dispatched and defined outside of the original widget file, i think we should find a new name for it, it became something else.

Reply:
Oh yeah I meant that the Action itself is the same one. (And it's technically an action on a downloaded file either way). But I do like `ChatAction` 👍🏽, it's a bit more obvious